spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/SparkUtils.java

```diff
@@ -84,7 +85,7 @@ static void validateConfigParameters(Schema inputSchema, @Nullable String featur
       throw new IllegalArgumentException("Cannot specify values for both featuresToInclude and featuresToExclude. " +
                                            "Please specify fields for one.");
     }
-    Map<String, Integer> fields = new HashMap<>();
+    Map<String, Integer> fields = new LinkedHashMap<>();
```

Comment:
Can you explain the cause of the flakiness and how this change solves it? Since we iterate over [inputSchemaFields](https://github.com/caskdata/hydrator-plugins/blob/develop/spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/SparkUtils.java#L106) the same order as schema